tests/test_surface/test_file_io.py

```diff
@@ -116,10 +117,13 @@ def test_complex_io(surf, fformat, output_engine, input_engine):
         "irap_ascii",
         "irap_binary",
         "zmap_ascii",
+        "ijxyz",
     ]:
         pytest.skip("Only one engine available")
     if fformat == "petromod":
         pytest.xfail("Several hypotesis failures (4)")
+    if fformat == "ijxyz":
+        pytest.xfail("Several hypotesis failures (4)")  # expected
```

Comment:
I just copied the string from the petromod example, but not what `(4)` refers to...

Reply:
Removed